frontend/src/components/worksheets/items/SchemaItem/SchemaItem.js

```diff
@@ -208,6 +208,30 @@ class SchemaItem extends React.Component<{
         }
     }
 
+    componentDidMount() {
+        if (this.props.create) {
```

Comment:
Can you add a comment to this function explaining what this code is meant for?

Reply:
done